doc/modules/tree.rst

```diff
@@ -535,20 +535,26 @@ Log Loss or Entropy:
 
       \mathrm{LL}(D, T) = \sum_{m \in T} \frac{n_m}{n} H(Q_m)
 
+.. _hellinger_distance_mathematical_formulation:
+
```

Comment:
I think adding a label here might be too granular. If we do, we also need to add some for the other criteria.

Reply:
makes sense
removed